tango/release.py

```diff
@@ -49,7 +49,7 @@ class Release:
     version = '.'.join(map(str, version_info[:3]))
     release = ''.join(map(str, version_info[3:]))
     separator = '.' if 'dev' in release or 'post' in release else ''
-    version_long = version + separator + release
+    version_long = "9.2.3"
```

Comment:
Has this change been committed by mistake?

Reply:
I think I had a problem somewhere and tested it like that and it stayed,